### PR TITLE
add resourceGroup to Azure disk params

### PR DESCRIPTION
**Pull request by mcurtiscmu**
Move resource group to correct place, deleted extra word and space.

### CODE REVIEW

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -610,6 +611,7 @@ parameters:
   unmanaged disk in the same resource group as the cluster. When `kind` is 
   `managed`, all managed disks are created in the same resource group as 
   the cluster.
+* `resourceGroup`: Specify the resource group in which the Azure disk will be created. It must be an existing resource group name. If it is unspecified, the disk will be placed in the the same resource group as the current Kubernetes cluster.
```
Comment (sftim) [inline content/en/docs/concepts/storage/storage-classes.md]: This line is wrapped differently from the ones before it. I recommend following the wrapping that's already set.

Comment (jaypipes) [reply content/en/docs/concepts/storage/storage-classes.md]: Agreed with @sftim, would be best to follow the line-wrapping that is in place here.

Comment (mcurtiscmu) [reply content/en/docs/concepts/storage/storage-classes.md]: For sure, thank you!

Comment (sftim) [reply content/en/docs/concepts/storage/storage-classes.md]: @mcurtiscmu I wonder if you've got time to tweak this pull request?

**Comment by zacharysarah** [comment]
@mcurtiscmu Please feel free to `/reopen` this PR when you're ready to continue.

/close

**Pull request closed without merging**